<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityAlertDialog</class>
 <widget class="QDialog" name="SecurityAlertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonSecAlertHome">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>272</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>居家</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecAlertDisArm">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>336</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>撤防</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecAlertReturn">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>400</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonSecAlertOut">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>208</y>
     <width>180</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>外出</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfo">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>20</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>,white</family>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26px, white;
border: 0px;</string>
   </property>
   <property name="text">
    <string>报警录像 - 1号有线防区</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frameVideoShow">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>77</y>
     <width>518</width>
     <height>377</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QFrame" name="framePanel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>18</y>
     <width>538</width>
     <height>450</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <zorder>framePanel</zorder>
  <zorder>frameVideoShow</zorder>
  <zorder>pushButtonSecAlertHome</zorder>
  <zorder>pushButtonSecAlertDisArm</zorder>
  <zorder>pushButtonSecAlertReturn</zorder>
  <zorder>pushButtonSecAlertOut</zorder>
  <zorder>labelInfo</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonSecAlertOut</tabstop>
  <tabstop>pushButtonSecAlertHome</tabstop>
  <tabstop>pushButtonSecAlertDisArm</tabstop>
  <tabstop>pushButtonSecAlertReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
